pxtlib/tutorial.ts

```diff
@@ -305,6 +306,27 @@ ${code}
         return { header, hint, requiredBlocks };
     }
 
+    function categorizingValidationRules(listOfRules: pxt.Map<boolean>) {
+        const ruleNames: string[] = Object.keys(listOfRules);
+        let enabledRulesMessageArr: string[] = [];
+        let turnedOffRulesMessageArr: string[] = [];
```

Comment:
```suggestion
        const enabledRulesMessageArr = [];
        const turnedOffRulesMessageArr = [];
```

nit: preferable to have const for these & enabledRulesMap.

(maybe worth a sidenote, having these as let makes it feel like you're thinking of const as frozen / immutable -- only the original reference / pointer is immutable, so it always points to the same array but the contents of that array can be modified after setting a const)